### PR TITLE
ADD DF00 for mi:muz:tuch

### PR DESCRIPTION
**Pull request by tadfmac**
Hello, 

I'm developing open source USB-MIDI touch controller device named "mi:muz:tuch".
https://github.com/tadfmac/mi-muz/tree/master/applications/tuch

So, I want to VID/PID. (PID requested: 0xDF00)

I hope so you accept marge my pull request!

Best regards,
Hideyuki Kodama (D.F.Mac. @TripArts Music)

### CODE REVIEW

File: 1209/DF00/index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: pid
+title: mi:muz:tuch
+owner: D.F.Mac.@TripArts Music
```
Comment (Arachnid) [inline 1209/DF00/index.md]: The 'owner' field needs to match the directory name of your org - in this case, 'TripArts-Music'.

**Comment by Arachnid** [comment]
Nice project! Apologies for the delay responding. If you can fix the owner issue, I'll be happy to approve the PR.

**Comment by Arachnid** [comment]
Actually, one other request: Can you please upload or link to the hardware source files? You specify an open-source license for them, but they're not included anywhere I can see.

**Comment by tadfmac** [comment]
Sorry, I shared eagle-files (.sch .pro .brd) on project repository now.
Could you check it again, please?

**Comment by Arachnid** [comment]
Looks good - you still need to fix the 'owner' field, though.

**Comment by tadfmac** [comment]
oh.. OK, I fixed owner.

**Comment by Arachnid** [comment]
One last thing - please delete `.DS_Store` from your PR.

**Comment by tadfmac** [comment]
oh... Sorry, I removed .DS_Store from my PR.

**Comment by Arachnid** [comment]
Thanks! Looks like an awesome bit of kit.

**Pull request merged**